source/val/validate_extensions.cpp

```diff
@@ -273,12 +287,14 @@ spv_result_t ValidateOperandDebugType(
 }
 
 spv_result_t ValidateClspvReflectionKernel(ValidationState_t& _,
-                                           const Instruction* inst) {
+                                           const Instruction* inst,
+                                           uint32_t version) {
+  const auto inst_name = ReflectionInstructionName(_, inst);
```

Comment:
nitpick.  I'd defer constructing the string until the time I know it's needed.
You can do that by making this a lambda, and then only calling it when generating the diagnostic.